packages/go/stbernard/analyzers/analyzers.go

```diff
@@ -17,60 +17,65 @@
-// Run all registered analyzers and collects the results into a CodeClimate-like JSON string
-//
-// If one or more entries have a severity of "error", this function will return a valid JSON string AND an error stating
-// that a high severity result was found
-func Run(cwd string, modPaths []string, jsPaths []string, env environment.Environment) (string, error) {
-	var (
-		severityError bool
-	)
+func outputSeverityMap(results codeclimate.SeverityMap, outputAllSeverity bool) error {
+	var exitErr error
 
-	golint, err := golang.Run(cwd, modPaths, env)
-	if err != nil {
-		return "", fmt.Errorf("golangci-lint: %w", err)
-	}
+	for _, nextSeverity := range results.SortedSeverities() {
+		isHighSeverity := nextSeverity.Priority() > codeclimate.SeverityError.Priority()
 
-	eslint, err := js.Run(jsPaths, env)
-	if err != nil {
-		return "", fmt.Errorf("eslint: %w", err)
-	}
+		if isHighSeverity {
+			exitErr = ErrSeverityExit
+		}
 
-	codeClimateReport := append(golint, eslint...)
+		if isHighSeverity || outputAllSeverity {
+			fileEntries := results[nextSeverity]
 
-	for idx, entry := range codeClimateReport {
-		// We're using err == nil here because we want to do nothing if an error occurs
-		if path, err := filepath.Rel(cwd, entry.Location.Path); err != nil {
-			slog.Debug("File path is either already relative or cannot be relative to workspace root", "err", err)
-		} else {
-			codeClimateReport[idx].Location.Path = path
-		}
+			// Issues of this severity should be output directly
+			for filePath, entries := range fileEntries {
+				fmt.Printf("%s\n", filePath)
 
-		if entry.Severity == "error" || entry.Severity == "major" || entry.Severity == "critical" || entry.Severity == "blocker" {
-			severityError = true
+				for _, entry := range entries {
+					fmt.Printf("\tLine %d - %s\n", entry.Location.Lines.Begin, entry.Description)
+				}
+			}
 		}
 	}
 
-	if jsonBytes, err := json.MarshalIndent(codeClimateReport, "", "    "); err != nil {
-		return "", fmt.Errorf("marshaling code climate report: %w", err)
-	} else if severityError {
-		return string(jsonBytes), ErrSeverityExit
-	} else {
-		return string(jsonBytes), nil
+	return exitErr
+}
+
+// Run all registered analyzers and collects the results into a CodeClimate-like structure
+//
+// If one or more entries have a severity of "error" this function returns an error stating
+// that a high severity result was found.
+func Run(paths workspace.WorkspacePaths, env environment.Environment, outputAllSeverity bool) error {
+	var exitErr error
+
+	if golintResults, err := golang.Run(paths.Root, paths.GoModules, env); err != nil {
+		return fmt.Errorf("golangci-lint: %w", err)
+	} else if err := outputSeverityMap(golintResults, outputAllSeverity); err != nil {
+		exitErr = err
+	}
+
+	if eslintResults, err := js.Run(paths.YarnWorkspaces, env); err != nil {
+		return fmt.Errorf("eslint: %w", err)
+	} else if err := outputSeverityMap(eslintResults, outputAllSeverity); err != nil {
+		exitErr = err
```

Comment:
This will overwrite the exitErr from golintResults, effectively making it a no-op.